data/other/logs/package/alternatives.yml

```diff
@@ -0,0 +1 @@
+log_path: /var/log/alternatives.log
```

Comment:
Uh wokay ... I'm not sure that this file really is any useful ? Can you elaborate on what kind of insight it provides ?

Reply:
Could be usefull in some situation with malicious programs running as default programs.
Maybe also to debug some php version conflict see https://github.com/YunoHost-Apps/framaforms_ynh/blob/testing/scripts/install#L191 .